app/src/main/java/com/github/sdpcoachme/database/FireDatabase.kt

```diff
@@ -259,5 +267,24 @@ class FireDatabase(databaseReference: DatabaseReference) : Database {
         }
         return future
     }
+
+    companion object {
+
+        /**
+         * Marks all messages in a list as read, except for the messages sent by the current user
+         *
+         * @param chat the chat whose messages to mark as read
+         * @param currentUserEmail the email of the current user
+         */
+        fun markOtherUsersMessagesAsRead(chat: Chat, currentUserEmail: String): Chat {
```

Comment:
This should probably be a method of Chat (would make more sens in my opinion)

Reply:
This will also be adapted